c-parser/c99.lua

```diff
@@ -64,7 +64,7 @@ defs["decl_ids"] = typed("string, number, table -> boolean, Decl?", function(_,
     -- store typedef
     if elem(decl.spec, "typedef") then
         if not (decl.ids and decl.ids[1] and decl.ids[1].decl) then
-            return false, nil
+            return true
```

Comment:
oh, btw, do you remember why was this bit necessary?

Reply:
The OSX headers have some typedefs that end up being defined twice, and returning nil in a match-time capture (the `=>`) makes the pattern fail.